accepted/00076-live-patching-eol-data.md

```diff
@@ -0,0 +1,119 @@
+- Feature Name: Live Patching EOL Date Information
+- Start Date: 2021-02-08
+- RFC PR:
+
+# Summary
+[summary]: #summary
+
+Enhance live patching integration by providing EOL information on installed patches.
+
+# Motivation
+[motivation]: #motivation
+
+SUSE only provides live patches for kernels up to 1 year after original release date of that kernel. It means that after one year, the user must reboot and use a new kernel so that they can keep getting live patches for another year.
+
+This feature is about enhancing the live patching experience in Uyuni by implementing the following improvements:
+
+1. Add information about the EOL of kernels in the UI
+2. Warn user when a live kernel is approaching EOL, or past EOL
+
+The existing live patching integration in Uyuni is described in the initial [Live Patching RFC](https://github.com/uyuni-project/uyuni-rfc/blob/master/accepted/00027-sle-live-patching.md).
+
+# Detailed design
+[design]: #detailed-design
+
+## Live patching EOL information
+
+SUSE provides lifecycle information on any product to users via 'zypper lifecycle' plugin. The actual data about a product is provided in a plain CSV file in a supplementary package per product. In case of live patching, this package is called `lifecycle-data-sle-live-patching` and is installed together with the product. Zypper lifecycle plugin reads this data to display human-friendly information to users.
+
+Uyuni shall use this data to provide the following:
+ - Display EOL dates for the current live patch in system details
+ - Display an out-of-support systems list in the home page
+ - Use the notifications system and/or send emails to report live patched systems approaching/past EOL
+ - The notifications/emails will be sent in advance according to a time value configurable from the UI
```

Comment:
- Provide EOL information via API (expand to what namespaces, probably later, after the SQL)